src/lib.ts

```diff
@@ -414,3 +419,9 @@ export function getChoiceOption(partialText: string): number {
   }
   return -1;
 }
+
+export function customUserConfirm(msg: string, defaultValue: boolean, timeOut?: number): boolean {
```

Comment:
This function definitely should get renamed, although I'm not entirely sure to what.